### PR TITLE
Replaces Log with Timber

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
Replaces Log with Timber in FormChooserList class

**Edit:**
Replaces Log with Timber in classes in `activities` package

### CODE REVIEW

**Comment by lognaturel** [comment]
👍 You can do a few classes in this PR if you want and just let us know when you're stopping so someone can merge. 😄 That way others can easily see the current status.

**Comment by shobhitagarwal1612** [comment]
Sure

**Comment by shobhitagarwal1612** [comment]
Please review these changes

File: collect_app/src/main/java/org/odk/collect/android/activities/FormHierarchyActivity.java [hunk 1]
```diff
@@ -361,7 +360,7 @@ public void refreshView() {
             // set the controller back to the current index in case the user hits 'back'
             formController.jumpToIndex(currentIndex);
         } catch (Exception e) {
-            Log.e(t, e.getMessage(), e);
+            Timber.e(e.getMessage(), e);
```
Comment (lognaturel) [inline collect_app/src/main/java/org/odk/collect/android/activities/FormHierarchyActivity.java]: Make sure to read the `Timber` docs carefully. This should just be `Timber.e(e)`!

**Comment by lognaturel** [comment]
Cool. Can you please use format strings instead of concatenation when you can? I think there are just a few instances and you can find them in the diff by searching for `" +`.

**Comment by shobhitagarwal1612** [comment]
Done

**Pull request merged**